plugins/com.google.cloud.tools.eclipse.appengine.validation/plugin.xml

```diff
@@ -118,7 +118,7 @@
 		  scope="total"
 		  class="com.google.cloud.tools.eclipse.appengine.validation.AppEngineWebXmlSourceValidator"
 		  id="appEngineWebXmlSourceValidator">
-		<contentTypeIdentifier id="appengine-web.xml">
+		<contentTypeIdentifier id="com.google.appengine.dd.xml">
```

Comment:
When I proposed this to @abbiefarr I thought it would be good to have a similar name for the `app.yaml` file.  So would `com.google.appengine.conf.xml` and `com.google.appengine.conf.yaml` seem ok?